harvard-lts/etd_dash_service
# Etd 160

**Adding dash scripts, phase 1**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/ETD-160)

# What does this Pull Request do?
In this pr, existing scripts are added to the docker, refactored as necessary. Also includes telemetry tracing Test coverage above 80 currently, we can revisit 100% threshold if we can get the pragma skipping to work with flake8. Still missing from this, coming in next phase, is the job monitor (jobmon statements left in as comments fpr when util classes in place)

# How should this be tested?
Clone this repo and check out this branch
Follow readme for setting up telemetry prereq, starting up local docker.
Exec in to docker and run pytest
Once in qa, we can test a submission (has been done already on dev)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Y
- integration tests? Y

# Interested parties
Tag (@ mention) interested parties
